tests/test_maintracker.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+import os
+import json
+import psycopg2
+from flask.testing import FlaskClient
+
+
+conn = psycopg2.connect("dbname='m_tracker_test' user='mmosoroohh' host='localhost' password='test123'")
+cur = conn.cursor()
+
+from app.app import create_app
+
+class MaintenanceTrackerTestCase(unittest.TestCase):
+    """This class represent user signup, signin, make request"""
+
+    def setUp(self):
+        """Define test variable and initialize app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client()
+        self.request = {'name': 'Computer Monitor', 'description': 'Broken screen', 'category': 'repair', 'department': 'Finance'}
+
+        # bind the app to the current context
+        with self.app.app_context():
+            # create all tables
+            cur.execute('''CREATE TABLE IF NOT EXISTS requests(id serial PRIMARY KEY, name varchar, description varchar, category varchar, department varchar, user_id INT REFERENCES(id));''')
```

Comment:
This could be extracted into a single function that can be called instead of pasting the code